modules/loader.js

```diff
@@ -436,44 +436,51 @@ __private.loadBlockChain = function () {
 		}
 	}
 
-	library.db.task(checkMemTables).then(function (results) {
-		var count = results[0].count;
+	library.db.task(checkMemTables).then(function (res) {
+		var countBlocks           = res[0],
+			getGenesisBlock       = res[1],
+			countMemAccounts      = res[2],
+			validateMemBalances   = res[3],
+			dbRoundsExceptions    = res[4];
 
+		var count = countBlocks.count;
 		library.logger.info('Blocks ' + count);
 
-		var round = modules.rounds.calc(count);
+		var round = slots.calcRound(count);
 
 		if (count === 1) {
 			return reload(count);
 		}
 
-		matchGenesisBlock(results[1][0]);
+		matchGenesisBlock(getGenesisBlock[0]);
 
 		verify = verifySnapshot(count, round);
 
 		if (verify) {
 			return reload(count, 'Blocks verification enabled');
 		}
 
-		var missed = !(results[2].count);
+		var missed = !(countMemAccounts.count);
 
-		if (missed) {
-			return reload(count, 'Detected missed blocks in mem_accounts');
-		}
+		// if (missed) {
+		// 	return reload(count, 'Detected missed blocks in mem_accounts');
+		// }
```

Comment:
It's commented temporarly, but it will be removed together with corresponding field in `mem_accounts`.